resources/views/platform/beaten-games-leaderboard-page.blade.php

```diff
@@ -18,6 +19,7 @@
     <x-beaten-games-leaderboard.meta-panel
         :allSystems="$allSystems"
         :gameKindFilterOptions="$gameKindFilterOptions"
+        :leaderboardKind="$leaderboardKind"
```

Comment:
There seems to be a slight discrepancy between the aggregated and non-aggregated data.

master:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/19508f72-faad-4970-ac11-4e4d34b59666)

this branch:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/b38958c6-5834-456b-a3da-d2887ea0e4f6)

Reply:
I noticed this as well. I believe the latest developer dumps have some discrepancies between `player_games` and `SiteAwards`.

In https://github.com/RetroAchievements/RAWeb/pull/1999/commits/e4c2e5e5554d5a0669654338b46bf5dbe3676ac0#diff-d1cd771d569c8de22cd2a5fbb203f1679cdd74af5e55f288f169697ed8bbeff8, I changed `UpdatePlayerRanks` to use `SiteAwards` instead of `player_games` before building aggregates. It occurred to me that `SiteAwards` is likely the better source of truth for building these rankings. I believe it's possible I pushed this commit up right after you did a full sync operation.